ocs_ci/deployment/tnf.py

```
@@ -0,0 +1,543 @@
+                base_ocp = BaseOCPDeployment()
+                base_ocp.test_cluster()
+
+                logger.info("OCP cluster deployed via dev-scripts on EC2 hypervisor")
+            except Exception:
+                logger.error(
+                    "Hypervisor deployment failed. Instance will be "
+                    "preserved for debugging. Run --teardown to clean up."
+                )
+                raise
+
+        def _get_hypervisor(self):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_get_hypervisor`

(ANN202)